pypeit/tracepca.py

```diff
@@ -166,7 +243,7 @@ def build_interpolator(self, order, ivar=None, weights=None, function='polynomia
         self.upper = upper
         self.maxrej = maxrej
         self.maxiter = maxiter
-        self.pca_bpm, self.fit_coeff, self.minx, self.maxx \
+        self.pca_bpm, self.fit_coeff, self.minx, self.maxx, self.pypeitFits \
```

Comment:
fit_coeff, minx, and maxx need not be returned now, since that info is in pypeitFits. See previous comment.

Reply:
@kbwestfall